src/elements/kw-view-feed/kw-view-feed.html

```diff
@@ -148,17 +167,33 @@
             <div id="detail">
                 <div class="details">
                     <button type="button" class="back-button" on-tap="_backToFeed">Back to feed</button>
-                    <kw-share-detail share="[[selectedShare]]"
-                                        tombstone$="[[!selectedShare]]"
-                                        on-post-comment="_onPostComment"></kw-share-detail>
+                    <kw-share-detail 
+                    share="[[selectedShare]]"
+                    tombstone$="[[!selectedShare]]"
+                    on-post-comment="_onPostComment"
+                    on-remix-action="_onRemixAction"> 
+                    </kw-share-detail>
                 </div>
             </div>
         </div>
+        <kano-modal id="shopDialog" assets-path="../../bower_components/web-components/demo-assets" banner-link="../../bower_components/web-components/demo-assets/avatar/judoka-face.svg">
+            <iron-image sizing="contain" src="http://world.kano.me/assets/projects/overlay-photo.png"></iron-image>
+            <h2>Computer Kit Exclusive</h2>
+            <p>Tame the world of Minecraft, build houses, crush mountains with just a few lines of code. Only on the Computer Kit.</p>
+            <hr>
+            <a href="http://kano.me"><button>Shop now</button></a>
+        </kano-modal>
+        <kano-modal id="loaderDialog" assets-path="../../bower_components/web-components/demo-assets" banner-link="../../bower_components/web-components/demo-assets/avatar/judoka-face.svg">
```

Comment:
Could we move these to the main assets folder for the app and reference there? We might not always keep these demo assets in the web-components repo, or expect any external dependencies on them, so good to ensure that they're integrated with the app.